Scripts/Generator/Room.cs

```diff
@@ -28,6 +32,23 @@ public Room(Vector2[][] polygon, int unitSize, Material material)
 
         AddChild(floorMesh);
         Translation = new Vector3(0, -unitSize, 0);
+
+        floorMesh.UseCollision = true;
+        floorMesh.CollisionLayer = 1;

```

Comment:
`floorMesh.CollisionLayer = ColLayer.Environment;`

Reply:
I alr forgot got this